src/Draco.Compiler/Internal/Symbols/WellKnownTypes.cs

```diff
@@ -45,6 +142,12 @@ internal sealed partial class WellKnownTypes
                 m.Name == "Format"
              && m.Parameters is [_, { Type: TypeInstanceSymbol { GenericDefinition: ArrayTypeSymbol } }]));
     private MetadataMethodSymbol? systemString_Format;
+
+    public TypeSymbol InstantiateArray(TypeSymbol elementType, int rank = 1) => rank switch
+    {
+        1 => this.ArrayType.GenericInstantiate(elementType),
+        int n => new ArrayTypeSymbol(n, this.SystemInt32).GenericInstantiate(elementType),
+    };
```

Comment:
Why is this in the Singletons region?

Reply:
It's an oversight. I'll fix it tonight.

Reply:
It's not in the Singletons region but the Methods region.